integration/experiment/energy_efficiency/Makefile.mk

```diff
@@ -36,6 +36,7 @@ EXTRA_DIST += integration/experiment/energy_efficiency/barrier_frequency_sweep.p
               integration/experiment/energy_efficiency/run_barrier_frequency_sweep_nekbone.py \
               integration/experiment/energy_efficiency/run_power_balancer_energy_dgemm.py \
               integration/experiment/energy_efficiency/run_power_balancer_energy_nasft.py \
+              integration/experiment/energy_efficiency/run_power_balancer_energy_hpcg_mkl.py \
```

Comment:
this one is in the other PR; I renamed it to run_power_balancer_energy_hpcg.py now that there's only one version